BSP/Inc/BSP_Contactors.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __BSP_CONTACTORS_H
+#define __BSP_CONTACTORS_H
+
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+void BSP_Contactors_Init(void);
+void read_Contactor_States(int16_t *contactorStates);
+void BSP_Contactor_Set(int16_t motorContactorState, int16_t arrayContactorState);
+
```

Comment:
Overall, this file needs documentation (see the style guide or other modules for examples). Also, the parameter types don't need to be int16_t since a contactor state is only ever a 0 or 1. In fact, I created an enum in config.h that defines an ON and OFF state for the switches, and that can be used here as well. It just makes it easier to work with.

Reply:
Added documentation for all functions in BSP_Contactors.h. Implemented enum from config.h into the set function.